mod/settings.php

```diff
@@ -1277,7 +1277,7 @@ function settings_content(App $a) {
 
 		'$h_pass' 	=> t('Password Settings'),
 		'$password1'=> array('password', t('New Password:'), '', ''),
-		'$password2'=> array('confirm', t('Confirm:'), '', t('Leave password fields blank unless changing')),
```

Comment:
Sorry, I assumed this was redundant... I wonder was in the past the changing password section linked to any of the other sub-sections in these settings? So that for example in the past someone wanted to change the time zone and thought the had to enter their password? 

Currently, in IMO, this hint looks very much out of place in this section that is purely for changing the password. The hint, I felt, almost gives the impression that the user has to enter the current (old) password, but should leave the new one empty if they don't want to change their password. But this of course would than be illogical.

If people think this hint should be remain, I'm happy to amend this PR. Alternatively to make it even more clear, we also could change the section heading to "Password Change" as it is already done in the two English localisations.

Either way not a problem. Thanks for coordinating this so smoothly!